sound/voice_data.h

```diff
@@ -15,7 +15,6 @@
 // They also have good expression.
 // 
 #ifdef FEMALE_TALKIE_VOICE
-const uint8_t spBATTERYLVL[] PROGMEM = {0xa8,0xf3,0xc2,0xdd,0xc3,0x93,0xa8,0xb6,0x69,0x17,0x0b,0xb7,0xdd,0xac,0xac,0x35,0xaa,0x85,0xf6,0xb0,0xab,0xb1,0xa8,0x11,0xda,0xcb,0x29,0x36,0xa3,0x4a,0x65,0x2f,0xa7,0x84,0xf4,0x2a,0x75,0xb2,0x9c,0xea,0xd3,0xb3,0x4d,0xce,0x72,0xaa,0x0b,0x8b,0x76,0x39,0xc3,0x6d,0x2a,0xdc,0x32,0xa4,0x0c,0xb7,0xaa,0x30,0xcb,0x88,0xd2,0x9c,0xea,0x54,0x2a,0x22,0x6c,0x71,0x4b,0x42,0x1b,0xcb,0x30,0xc5,0x29,0x01,0xcb,0xb2,0x65,0x37,0xa7,0x07,0x4c,0x1d,0x93,0xdd,0xdc,0xe1,0xd0,0x63,0x5c,0x6e,0x72,0x0b,0xa7,0x9e,0x61,0xbb,0xce,0xd3,0xce,0xaa,0x57,0x2d,0xbb,0xc0,0xf2,0x6e,0xcb,0x88,0x14,0x02,0xaf,0xcc,0xd3,0x7c,0xb1,0x8b,0xac,0x6a,0xb1,0xf6,0xda,0x21,0xd4,0xbe,0x34,0xa2,0x43,0x97,0x20,0xfa,0xb2,0x1a,0xb4,0x3b,0xa2,0x6c,0xc7,0x63,0x45,0xc9,0x88,0xb3,0x9b,0xf0,0x51,0x36,0x23,0xce,0x31,0xbd,0x56,0x54,0xb7,0x24,0xfb,0x72,0x5f,0x63,0x62,0xd2,0x68,0xca,0xb9,0x9c,0xb6,0x8b,0x9a,0x08,0xd7,0x30,0x47,0x29,0x4c,0x7a,0x1d,0xbb,0x65,0xa7,0x30,0xaa,0x73,0x98,0x96,0x1c,0x82,0xa0,0xcf,0x60,0x8b,0xb6,0x0b,0x9c,0x5d,0xe5,0x8e,0x58,0x22,0xf6,0xa1,0x44,0x3a,0x23,0x03,0xc0,0x03};
```

Comment:
Still removing this?

Reply:
fixed

Reply:
Looks like spBATTERYLEVEL and spBATTLELEVELCURR are still being deleted?

Reply:
Yes, unless you prefer them in. 
I couldn't pull off a good sounding Male BATTLVLCURR, so there's only a female version. 
Neither that nor BATTERYLVL are not used anywhere, so I figure yes, delete them.

Reply:
That's fine, but it looks weird when you do it in an unrelated PR, and the description of the PR doesn't mention it.